docs/src/modules/utils/parseMarkdown.js

```diff
@@ -250,7 +251,7 @@ ${headers.components
               `<h${level}>`,
               `<a class="anchor-link" id="${hash}"></a>`,
               headingHtml,
-              `<a class="anchor-link-style" aria-hidden="true" aria-label="anchor" href="#${hash}">`,
+              `<a class="anchor-link-style" aria-hidden="true" href="#${hash}">`,
```

Comment:
The label is still present in the API pages: https://github.com/mui-org/material-ui/blob/d0a6b02aa88838fdb9da1489cc1cc9ea5cbd4a6d/docs/src/modules/components/ApiPage.js#L150

I added it to avoid the noise from WAVE:

<img width="826" alt="Screenshot 2021-03-26 at 00 11 00" src="https://user-images.githubusercontent.com/3165635/112555159-eda5b380-8dc7-11eb-8fbf-a3617571570b.png">

What do you think about the solution of https://docusaurus.io/?

<img width="635" alt="Screenshot 2021-03-26 at 00 12 45" src="https://user-images.githubusercontent.com/3165635/112555213-09a95500-8dc8-11eb-9bb7-f1dfb6888661.png">

Reply:
`title` is most of the time bad. This would be one of them. Also note the `aria-hidden`.

Reply:
> The label is still present in the API pages:

The ApiPage was added after this change. So it's not "still" but "again". We can cherry-pick [`267e78d` (#23791)](https://github.com/mui-org/material-ui/pull/23791/commits/267e78ddf9dbc57ed3e705238ad63a45fdb6a715) into ApiPage.